src/externalAPIs/search_ch.rb

```diff
@@ -1,6 +1,17 @@
-class SearchCH
-  def get_stationboard(station, date)
-    raise NotImplementedError, 'Method not implemented'
+require_relative 'external_api'
+require_relative 'models/stationboard_response'
+
+class SearchAPI < ExternalAPI
+  base_uri 'search.ch'
+
+  def initialize(options)
+    super(options)
+    @options = { query: { show_tracks: "1", show_subsequent_stops: "1", time: '00:01' } }
```

Comment:
options aren't the same when we call for different data. For example if you get a station it won't have time at all. The options should only be in the methods which will have options alredy set if needed